plugins/package-managers/cargo/src/main/kotlin/Cargo.kt

```diff
@@ -121,7 +122,7 @@ class Cargo(
 
             else -> {
                 contents.metadata.mapNotNull { (k, v) ->
-                    (v as? String)?.let { k.unquote().removePrefix("checksum ") to v }
+                    k.unquote().withoutPrefix("checksum ")?.let { it to v }
```

Comment:
What was the purpose of checking if `v` is a string and why is it not required anymore?

Reply:
This is a fixup for the migration done in 44523e412a504651cfad2f2617653818df7cc4a8 which did not really make sense to split out into a preceding commit (as I would then change `map {}` <-> `mapNotNull {}` and back again). I've clarified this in the commit message.